source/blog/2016-10-28-next-london-2016-google-cloud-platform-s-user-conference.md

```diff
@@ -2,9 +2,9 @@
 date: '2016-10-28 10:39 +0100'
 published: true
 title: NEXT London 2016 - Google Cloud Platform's User Conference
-authors: 
+author: Charlie Egan
+coauthors:
```

Comment:
The reason I changed it from the initial author and authors is that I didn't want to have them being one or the other. But agree that if we remove the author and just go with authors consistently that will be good.